decryption_utils/Go/decrypt.go

```diff
@@ -66,13 +77,13 @@ func decryptFile(privateKey *rsa.PrivateKey, encryptedKey []byte, filename strin
 	decryptedKey, err := rsa.DecryptOAEP(
 		sha256.New(), rand.Reader, privateKey, encryptedKey, []byte(base))
 	if err != nil {
-		fmt.Println("Failed to decrypt encrypted key")
+		fmt.Println("Failed to decrypt encrypted key. Error:", err)
 		os.Exit(3)
```

Comment:
Is there value in returning different error codes now that we are properly reporting the exact error?  Does it make sense to switch it back to returning the standard exit code of 1 in the event of an error?

Reply:
Is there any chance we'd want to handle each exit code differently when the decryptor is called from bcda_client.go? I can't think of any reason to when we're printing the errors, so I agree that it makes sense to set them all to 1.

Reply:
Agree!